artifacts/definitions/Windows/Applications/Chrome/History.yaml

```diff
@@ -31,11 +33,16 @@ sources:
 
         SELECT * FROM foreach(row=history_files,
           query={
-            SELECT User, FullPath,
-                   Mtime,
-                   visited_url,
-                   title, visit_count, typed_count,
-                   timestamp(winfiletime=last_visit_time * 10) as last_visit_time
+            SELECT User, 
+                   id AS url_id,
+                   timestamp(epoch=((visit_time / 1000000) - 11644473600)) AS visit_time,
```

Comment:
you can just do `timestamp(winfiletime=visit_time)`

Reply:
Oh thats awesome. I'll test that now

Reply:
Thats worked a treat, thanks for that. I didn't realise a chrome timestamp was the same as that. Fixed in [ff7e381](https://github.com/angry-bender/velociraptor/commit/ff7e38115356d6c5f18a80171c47e120c823d3f6)

![image](https://user-images.githubusercontent.com/18164137/152289106-2ee32fe7-0cba-4691-ab49-65d581b2d47c.png)